independent-projects/resteasy-reactive/client/processor/src/main/java/org/jboss/resteasy/reactive/client/processor/scanning/ClientEndpointIndexer.java

```diff
@@ -61,7 +61,7 @@ public ClientEndpointIndexer(AbstractBuilder builder, String defaultProduces, bo
     }
 
     public MaybeRestClientInterface createClientProxy(ClassInfo classInfo,
-            String path) {
+            String path, boolean removesTrailingSlash) {
```

Comment:
This logic would be easier to follow if the path is already updated with the start and trailing slashes logic. Wdyt?

Reply:
Sure